src/effects.cpp

```diff
@@ -258,7 +258,6 @@ std::shared_ptr<LEDStripEffect> GetSpectrumAnalyzer(CRGB color)
 #define STARRYNIGHT_PROBABILITY 1.0
 #define STARRYNIGHT_MUSICFACTOR 1.0
 
-DRAM_ATTR std::unique_ptr<EffectFactories> g_ptrEffectFactories = nullptr;
 std::map<int, JSONEffectFactory> g_JsonStarryNightEffectFactories =

```

Comment:
Suggestion to move g_ptrEffectFactories back to here:

```suggestion
DRAM_ATTR std::unique_ptr<EffectFactories> g_ptrEffectFactories = nullptr;

std::map<int, JSONEffectFactory> g_JsonStarryNightEffectFactories =
```